src/views/RegisterUser.vue

```diff
@@ -151,27 +151,68 @@ export default defineComponent({
           },
           confirm: {
             requiredValidator,
-
-            sameAs: sameAs(state.password.password),
+            sameAs: sameAs(state.formData.password.password),
           },
         },
         registrationKey: {
           requiredValidator,
-          keyValidator,
         },
       };
     });
-    const v$ = useVuelidate(rules, state);
+    const v$ = useVuelidate(rules, state.formData);
+    async function sendInfo() {
+      const userData: UserStateInterface['data'] = {
+        registrationToken: state.formData.registrationKey,
+        firstName: state.formData.firstName,
+        lastName: state.formData.lastName,
+        email: state.formData.email,
+        password: state.formData.password.confirm,
+        contacts: [{ id: Date.now(), type: 'email', main: false, email: state.formData.email }],
+      };
+      await store.dispatch('userStore/SET_USER_INFO', userData);
+    }
+    const showInfo = (status: string, message: string) => {
+      toast.add({ severity: status, summary: message, life: 6000 });
+    };
+    const resetFields = () => {
+      state.formData.firstName = '';
+      state.formData.middleName = '';
+      state.formData.lastName = '';
+      state.formData.email = '';
+      state.formData.password = {
+        password: '',
+        confirm: '',
+      };
+      state.formData.registrationKey = '';
+    };
+
+    async function showStatus() {
```

Comment:
seems this method is used as submit, but better to divide and make more logical function calls

create method which called `onSubmit`- which will contain method `sendInfo` and `showStatus`